extensions/amp-brightcove/0.1/test/test-amp-brightcove.js

```diff
@@ -133,14 +135,27 @@ describes.realWin(
             '/index.html?videoId=ref:amp-test-video&playsinline=true'
         );
 
+        const newSrc = new Promise((resolve) => {
```

Comment:
Since you changed `layoutCallback()` you're going to have to add this to `beforeEach()` for the whole test file. That's probably why the existing tests are failing when you had `return getIframeSrc_()....`